package.json

```diff
@@ -63,12 +65,15 @@
   },
   "files": [
     "bin",
-    "dist",
+    "dist/lib",
+	"dist/index.d.ts",
+	"dist/index.js",
+	"dist/index.js.map",
```

Comment:
Couldn't we just keep it as `dist/` instead? I think that'd be better to avoid anyone running into an issue in the future. At the very least, if what you're trying to do is ignore `dist/test/`, we _could_ do the `tsconfig.release.json` idea.